R/build.r

```diff
@@ -38,7 +43,8 @@ build_site <- function(pkg = ".",
                        with_vignettes = TRUE,
                        with_demos = TRUE,
                        with_readme = TRUE,
-                       launch = interactive()
+                       launch = interactive(),
+                       tracking_id = NULL
```

Comment:
Why not just call this `ganalytics_id` so you only need one variable? (Not both `tracking_id` and `ganalytics`?  That would eliminate most of the code below.